openshift/topics/introduction.adoc

```diff
@@ -1,35 +1,66 @@
-== Introduction
+== Introduction to Red Hat for Single Sign-On for OpenShift Image
 
 === What Is {project_name}?
+[role="_abstract"]
 {project_name} is an integrated sign-on solution available as a Red Hat JBoss Middleware for OpenShift containerized image. The {project_openshift_product_name} image provides an authentication server for users to centrally log in, log out, register, and manage user accounts for web applications, mobile applications, and RESTful web services.
 
-{openshift_name} is only available on the following platforms: *{openshift_image_platforms}*. For other available platforms, see link:{openshift_link_other}[{openshift_name_other}].
+{openshift_name} is available on *{openshift_image_platforms}*. For other available platforms, see link:{openshift_link_other}[{openshift_name_other}].
+
+=== Comparison: {project_openshift_product_name} Image versus Red Hat Single Sign-On
+The {project_openshift_product_name} image version number {project_version} is based on {project_name} {project_version}. There are some differences in functionality between the {project_openshift_product_name} image and {project_name}:
+
+The {project_openshift_product_name} image includes all of the functionality of {project_name}. In addition, the {project_name}-enabled JBoss EAP image automatically handles OpenID Connect or SAML client registration and configuration for *_.war_* deployments that contain *<auth-method>KEYCLOAK</auth-method>* or *<auth-method>KEYCLOAK-SAML</auth-method>* in their respective *web.xml* files.
 
 [[sso-templates]]
-Red Hat offers multiple OpenShift application templates utilizing the {project_openshift_product_name} image version number {project_version}. These define the resources needed to develop {project_name} {project_version} server based deployment and can be split into the following two categories:
+=== Templates for use with this software
+
+Red Hat offers multiple OpenShift application templates using the {project_openshift_product_name} image version number {project_version}. These templates define the resources needed to develop {project_name} {project_version} server based deployment. The templates can mainly be split into two categories: passthrough templates and reencryption templates. Some other miscellaneous templates also exist.
 
 [[passthrough-templates]]
-* Templates using HTTPS and JGroups keystores and a truststore for the {project_name} server, all prepared beforehand. These secure the TLS communication using link:{ocp311docs_passthrough_route_link}[passthrough TLS termination]:
+==== Passthrough templates
+
+These templates use HTTPS and JGroups keystores and a truststore for the {project_name} server.  They secure the TLS communication using passthrough TLS termination.
```

Comment:
replace `use HTTPS and JGroups` with `require HTTPS, JGroups keystores ... {project_name} server to exist beforehand`

Like mentioned in other comment above, all of them use HTTPS, JGroups keystores & RH-SSO truststore. But the difference is the passthrough ones require the user to provide the information / details about them, while the re-encrypt (x509) ones auto-generate them.